googleapiclient/discovery_cache/file_cache.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2014 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""File based cache for the discovery document."""
```

Comment:
It looks like the file used by the Cache implementation in this class changes from instance to instance - is that right? In that case, why isn't it an in-memory cache? Is it because the content is expected to be larger than would fit in memory? If so, please say so in either the module or class doc string.